tests/unit/test_utils.py

```diff
@@ -309,3 +312,20 @@ def test_get_call_centre_number(self):
         self.assertEqual(built_ew, expected_ew)
         self.assertEqual(built_cy, expected_cy)
         self.assertEqual(built_ni, expected_ni)
+
+    def after_census_day(self, year, month=None, day=None, hour=0, minute=0, second=0):
+        mocked_now_utc = datetime.datetime(year, month, day, hour, minute, second, 0)
+        with mock.patch('app.utils.View.get_now_utc') as mocked_get_now_utc:
+            mocked_get_now_utc.return_value = mocked_now_utc
+            self.assertTrue(View.check_if_after_census_day())
+
+    def before_census_day(self, year, month=None, day=None, hour=0, minute=0, second=0):
+        mocked_now_utc = datetime.datetime(year, month, day, hour, minute, second, 0)
+        with mock.patch('app.utils.View.get_now_utc') as mocked_get_now_utc:
+            mocked_get_now_utc.return_value = mocked_now_utc
+            self.assertFalse(View.check_if_after_census_day())
+
+    def test_check_before_after_census_day(self):
```

Comment:
Are any of these attributes that are set-up used anywhere? All changes to this file could be removed?

Reply:
Don't understand the comment

Reply:
All changes to test_utils.py appear to be redundant code. They are not called and could be removed with no effect on running the unit tests?